runtime/doc/api.txt

```diff
@@ -1737,7 +1737,7 @@ nvim__buf_stats({buffer})                                  *nvim__buf_stats()*
                 TODO: Documentation
 
                                                     *nvim_buf_add_highlight()*
-nvim_buf_add_highlight({buffer}, {src_id}, {hl_group}, {line}, {col_start},
+nvim_buf_add_highlight({buffer}, {ns_id}, {hl_group}, {line}, {col_start},
```

Comment:
change it in `src/nvim/api/buffer.c` so it won't get erased on next sync.

Reply:
Done. :)